projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts

```diff
@@ -135,7 +135,7 @@ describe('CaseFileViewFolderComponent', () => {
     expect(component.nestedDataSource).toEqual(treeDataSortedAlphabeticallyDesc);
   });
 
-  it('should set mediaViewer localStorage' +
+  it('should set mediaViewer localStorage ' +
```

Comment:
@MateiHMCTS - not sure why the extra space at the end...?

Reply:
needs a space between that row and the next one

Reply:
Ah - yes, sorry... it does!